Docs: Update azuread docs to mention about env variables (#39203)
grafana/grafana: docs/sources/auth/azuread.md

allowed_groups =
```

> You can also use the following enviroment variables to configure **client_id** and **client_secret**:

Reviewer: ```suggestion
You can also use these enviroment variables to configure **client_id** and **client_secret**:
```

```
GF_AUTH_AZUREAD_CLIENT_ID
GF_AUTH_AZUREAD_CLIENT_SECRET
```

> 

Reviewer: ```suggestion
```
